Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -1260,6 +1263,65 @@ private static void OnIsValidPropertyChanged(DependencyObject d, DependencyPrope
             }
         }
 
+        // Field used for accessing the ItemsSource though the interface ISupportIncrementalLoading
+        private ISupportIncrementalLoading _incrementalItemsSource;
+
+        private bool _canLoadIncrementalData;
+
+        private bool _itemsSourceIsIncremental;
```

Comment:
No need for this variable. Can just check if `_incrementalItemsSource` is not null

Reply:
Sure!